pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -294,7 +294,8 @@ Servers
       address="IP:PORT",     -- IP and PORT of the backend server (mandatory)
       qps=NUM,               -- Limit the number of queries per second to NUM, when using the `firstAvailable` policy
       order=NUM,             -- The order of this server, used by the `leastOustanding` and `firstAvailable` policies
-      weight=NUM,            -- The weight of this server, used by the `wrandom` and `whashed` policies
+      weight=NUM,            -- The weight of this server, used by the `wrandom` and `whashed` policies, default: 1
+                             -- Supported values are a minimum of 1, and a maximum of 2147483647.
```

Comment:
Technically a `signed int` maximum value could be as small as `32767`, but let's assume we are not going to run on 16-bit.

Reply:
Yeah, the actual check is pulling the max dynamically, but I went with the common value for documentation here. If someone happens to run on 16-bit and tries to add something larger than it, at least the error log will say the correct max.